sdks/go/node/containerruntime/docker/isGpuSupported.go

```diff
@@ -54,6 +54,9 @@ func isGpuSupported(
 		"",
 		noOpEventPublisher{},
 	)
+	if err != nil {
```

Comment:
this is the actual fix for the lack of rate limiting error surfacing during gpu testing